docs/user-guide/write-data.md

```diff
@@ -220,3 +220,41 @@ curl -X POST http://127.0.0.1:4000/v1/opentsdb/api/put -d '
 
 See [Prometheus Query Language](./prometheus.md#prometheus-query-language) to know how to write data.
 
+## Delete
+
+### SQL
+
+#### `Delete` Statement
+
+To delete a row from it by key `idc` and timestamp index `ts`:
+
+```sql
+DELETE FROM system_metrics WHERE idc = 'idc_a' and ts = 1667446797450;
+```
+
+```
+Query OK, 1 row affected (0.00 sec)
+```
+
+For more information about the `Delete` statement, please refer to the [SQL DELETE](/reference/sql/delete.md).
+
+#### HTTP API
+
+Use GET method to delete data:
```

Comment:
It is a minor anti-pattern for a `GET` method to have the ability to delete data from the server.